.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Release
+about: Create an issue for releasing new versions
+title: 'Release vX.X.X'
+labels: release
+assignees: ''
+
+---
+
+Like #4522, but for vX.X.X
```

Comment:
Should we not repeat the version that many times?

Reply:
Hovering the individual items will allow the release manager to create individual issues based on the item's name. So, keeping the version name there will make the version appear in the issue's title.

Reply:
Personally, I dislike creating separate issue per entry here, but it is just me, so keep it for the moment.